content/src/main/content/jcr_root/apps/acs-commons/components/content/shared-component-props-example/dialog.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Dialog"
+    title="Shared Component Properties Example - Local Properties"
+    xtype="panel">
+    <items jcr:primaryType="cq:WidgetCollection">
+        <title
+            jcr:primaryType="cq:Widget"
+            fieldLabel="Title"
```

Comment:
For consistency between the touch ui version and the classic versions, update this to be "Title Text"